### ISSUE TITLE
Tailon does not terminate properly tail command

### ISSUE THREAD

**Issue by espinraf** [gvalkov/tailon-legacy#26]
When a user close Tailon web page, tailon does not terminate the tail command properly, leaving <defunct> tail processes.

Tailon process: 
6667  6666 /usr/bin/python /usr/local/bin/tailon -a -f /var/log/.......

Defunct processes:

 ps -A -o pid,ppid,cmd | grep '<defunct>'
 7512  6667 [tail] <defunct>
12074  6667 [tail] <defunct>
12075  6667 [tail] <defunct>
19508  6667 [tail] <defunct>
23347  6667 [tail] <defunct>
24130  6667 [tail] <defunct>
24447  6667 [tail] <defunct>

**Comment by espinraf** [comment]
It seems if you add var.proc.wait() and the set to None  var.proc = None, it removes the defunct process

```
def killall(self):
    for i in ('tail', 'awk', 'grep', 'sed'):
        var = getattr(self, i)
        if var:
            log.debug('killing %s process: %s', i, var.pid)
            var.stdout.close()
            var.stderr.close()
            var.proc.kill()
            ##########
            var.proc.wait()
            var.proc = None
            ##########
            var = None
```

**Comment by gvalkov** [comment]
Hi and thanks for the catch. I'm in the middle of a heavy refactoring at the moment. I'll make sure to have this fixed when things settle down again.

**Comment by gvalkov** [comment]
Fixed in tailon 0.6.0. Again, thanks for the bug report.
